docs/_docs/Instructors/01-Teaching-process.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Teaching Process
+category: Instructors
+order: 1
+---
+
+Contents:
+- [Material structure](#materials)
+- [Areas of responsibility](#aors)
+- [Continuous improvement](#continuous-improvement)
+- [Timeline](#timeline)
+- [Key takeaways](#takeaways)
+
+---
+
+This page provides and overview of our teaching process and methodology. It is aimed at all volunteers
+that are involved or are thinking of getting involved in the process of building and maintaining our materials. 
+We start by describing lightly the structure and format of our materials; then we 
+describe how the different areas of responsibility interact, and which roles they play in 
+this process; finally, we provide an example timeline for the process.
+
+# Academy materials
+
+The product of our teaching process are the materials we hand out to the students.
+We follow the following format for those materials:
+
+* Jupyter notebooks: almost all content is delivered through jupyter notebooks. 
+This provides an interactive way of learning, as students can run code while reading
+through the explanations. 
+* Nbgrader: In the context of jupyter notebooks, we also perform evaluation through
+graded notebooks. For this purpose we use `nbgrader` (an overview of how to use it
+can be found in [Using nbgrader](./03-Using-nbgrader-for-Exercise-Notebooks.md).)
+
+All materials are planned, built and reviewed through github. In particular all repositories
+are found under the [Academy GitHub](https://github.com/LDSSA/). These are the main
+repositories to check and use:
+* [curriculum-development](https://github.com/LDSSA/curriculum-development) (public): Our curriculum is reviewed and updated in this repository 
+* [batchX-instructors](https://github.com/LDSSA/batch6-instructors) (private): To prepare the materials beforehand, instructors work in this
+private repository. It's in this repo that all review and discussion around specific materials 
+happens - for this purpose all feedback and suggestions should be posted as issues 
+* [batchX-students](https://github.com/LDSSA/batch5-students) (public):  Close to the deadline of the unit, we trigger a process that releases the content of 
```

Comment:
Same comment as above. I would remove the URL and add a phrase saying "Last edition repository can be found [here](https://github.com/LDSSA/batch5-students)" at the end